### ISSUE TITLE
request: make send mail popup about timezone configurable?

### ISSUE THREAD

**Issue by wivaku** [PrivateBin/PrivateBin#964]
## Steps to reproduce

When sending message as mail (#398) a popup appears:  
Recipient may become aware of your timezone, convert time to UTC?  
(select from: use current timezone, use UTC, close).

For unexperienced users this popup can cause confusion.

### What should happen
I would prefer this to be configurable. E.g. `mailtimezone = ask (default)/current/utc`.

## Basic information

**PrivateBin version:** 1.4.0

I can reproduce this issue on <https://privatebin.net>: Yes

**Comment by rugk** [comment]
Hmm possible, but note that it is a privacy feature/warning. As such maybe the UX/wording can be improved in general, if it is confusing?

**Comment by wivaku** [comment]
The warning applies only to the expiration timestamp, right? So:
`  - This link will expire after August 11, 2022 at 2:54:00 PM UTC.`
versus
`  - This link will expire after 8/11/2022, 4:54:00 PM.`  
(I assume local timezone is used, but no timezone is explicitly mentioned)

For my situation the concern is not a big one and does not justify a popup.  

Alternatively, make the configuration option e.g. `mailtimezone = ask (default)/utc`.
So, either ask or pick the safest option.

**Comment by rugk** [comment]
Yes that's what it is about. And no, the (red, danger) button should mention it uses the local time zone:
https://github.com/PrivateBin/PrivateBin/blob/c065a4b739bdad64d442dccf9ae4ef4ccc8ebf02/tpl/bootstrap.php#L157

UTC is indeed also possible, but could be confusing, I guess. So I am not against such an option. Also even if the server admin decides "local" option is fine, the user sees that in the mail client, so they can see what it exposes. The popup is, after all, just there to make users aware that this happens (when you just see a time, you may not get that this is a problem, after all).
I'm in favor for keeping `ask` the default though. And for documenting in the config that `local` is the unsafe option and is discouraged to be used.
